content/applications/productivity/discuss/overview/get_started.rst

```diff
@@ -1,87 +1,95 @@
 ========================
-Get Started with Discuss
+Get started with Discuss
 ========================
 
-Discuss allows you to bring all of your company’s communication together through messages, notes,
-and chat. Share information, projects, files, prioritize tasks, and stay connected with colleagues
-and partners throughout applications. Forge better relationships, increase productivity and
-transparency by promoting a convenient way of communicating.
+The *Discuss* app allows a company to bring all of their communication together through messages,
+notes, and chat. Users are able to share information, projects, and files. Prioritize tasks and stay
+connected with colleagues and partners across applications. Forge better relationships and increase
+productivity and transparency by promoting a convenient way of communicating.
 
 .. _discuss_app/notification_preferences:
 
-Choose your notifications preference
-====================================
+Choose notifications preference
+===============================
 
-Access your *Preferences* and choose how you would like your notifications to be handled.
+Access *preferences* by navigating to :menuselection:`Settings app --> Users --> User -->
+Preferences tab`. Then choose either :guilabel:`Handle by Emails` or :guilabel:`Handle in Odoo` for
+how the user's notifications should be handled.
 
-.. image:: get_started/change_preferences.png
+.. image:: get_started/preferences-user.png
    :align: center
-   :alt: View of the preferences page for Odoo Discuss
+   :alt: View of the Preferences tab for Odoo Discuss.
 
-| By default, the field is set as *Handle by Emails* making messages, notes, and notifications where
-  you were mentioned or that you follow, to be sent through email. By choosing *Handle in Odoo*,
-  they are shown in the *Inbox*.
-| Messages can then be *Marked as Todo*, *Replied*, or *Marked as Read*.
+By default, the :guilabel:`Notification` field is set as :guilabel:`Handle by Emails`. With this
+setting enabled, a notification email will be sent by Odoo every time a message is sent, a note is
+sent with an '@' mention, or a notification is sent for a record that the user follows.
```

Comment:
I suppose this would be user input, so its probably best to format it as such
```suggestion
sent with an `@` mention, or a notification is sent for a record that the user follows.
```